packages/fxa-settings/src/components/TextInput/index.tsx

```diff
@@ -74,6 +90,19 @@ export const TextInput = ({
           }}
         />
       </span>
+
+      {errorText && (
+        <span
+          data-testid="error-tooltip"
+          className={
+            errorTooltipClass
+              ? errorTooltipClass
+              : 'tooltip-text tooltip-showing bg-red-200 p-3 -mt-20 rounded text-sm'
+          }
+        >
+          {errorText}
+        </span>
+      )}
```

Comment:
It looks like this is going to run into a couple problems when the string is long.

1. If the text wraps, the container will flow overtop of the text input and prevent it from being interacted with. If we position the tooltip at the bottom edge of the input this will be less of a concern. I'm also unsure if it should be centered or left-aligned.
2. If the window width is large it'll stretch as far as it can. We might need a max-width wrapper.

![Screen Shot 2020-08-19 at 5 05 51 PM](https://user-images.githubusercontent.com/6392049/90689564-46174000-e23e-11ea-8773-16f687d7a649.png)

![Screen Shot 2020-08-19 at 5 06 33 PM](https://user-images.githubusercontent.com/6392049/90689814-a8704080-e23e-11ea-9824-7110a53b393d.png)

Reply:
I took this as a first run on the tooltips. per the jira issue:
> The PR for this should 1) create the input that we’re going to use for the secondary email because it’s a straightforward basic input (it’s fine for styles not to be totally yet)

https://jira.mozilla.com/browse/FXA-1648

I've created a follow up for styles, handling long text, and pulling out into a separate component https://jira.mozilla.com/browse/FXA-2451

Reply:
Ah okay cool - I didn't realize styles were coming later.